packages/adapters-library/src/adapters/morpho-blue/products/market-borrow/morphoBlueMarketBorrowAdapter.ts

```diff
@@ -17,7 +17,7 @@ export class MorphoBlueMarketBorrowAdapter extends MorphoBluePoolAdapter {
   getProtocolDetails(): ProtocolDetails {
     return {
       protocolId: this.protocolId,
-      name: 'MorphoBlue',
+      name: 'MorphoBlue Borrow',
```

Comment:
It wasn't possible to tell the difference in our UI, as they both had the same name.